addons/disposable/XEH_postInit.sqf

```diff
@@ -5,6 +5,7 @@ if (!hasInterface) exitWith {};
 
 ["inventoryDisplayLoaded", {[ACE_player, _this select 0] call FUNC(updateInventoryDisplay)}] call EFUNC(common,addEventHandler);
 ["playerInventoryChanged", {
-    [_this select 0, _this select 1 select 11] call FUNC(takeLoadedATWeapon);
-    [_this select 0] call FUNC(updateInventoryDisplay);
+    params ["_unit"];
+    [_unit, _this select 1 select 11] call FUNC(takeLoadedATWeapon);
```

Comment:
`_this select 1` is data in the format of what `EFUNC(common,getAllGear)` supplies.
Therefore `_this select 1 select 11` is the Classname as a STRING of the Launcher slot.